toonz/sources/toonz/tapp.cpp

```diff
@@ -675,14 +675,25 @@ void TApp::autosave() {
 
   if (scene->isUntitled()) {
```

Comment:
If you would like to automatically save only levels, scene can be kept untitled and not needed to be saved.